docs/openapi/components/schemas/common/FoodDefenseQuestion.yml

```diff
@@ -0,0 +1,53 @@
+$linkedData:
+  term: FoodDefenseQuestion
+  '@id': https://w3id.org/traceability#FoodDefenseQuestion
+title: Food Defense Question
+description: Information on a food defense inspection question and its response.
+type: object
+properties:
+  type:
+    oneOf:
+      - type: array
+        items:
+          type: string
+          enum:
+            - FoodDefenseQuestion
+      - type: string
+        const: FoodDefenseQuestion
+  number:
+    title: Number
+    description: An identifier for the question.
+    type: string
+    $linkedData:
+      term: number
+      '@id': https://schema.org/identifier
+  facility:
+    title: Facility
+    description: Must be one of (P) Processing Facility, or (B) Both Processing and Warehousing Facility.
```

Comment:
Just double-checking that it could not alternatively be `(W) Warehousing Facility`.

Reply:
That's correct